src/data-model-providers/codegen/CodegenDataModelProvider.cpp

```diff
@@ -14,6 +14,9 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+#include "app/ConcreteAttributePath.h"
+#include "app/util/attribute-metadata.h"
```

Comment:
Angle bracket includes like the other ones?  And not sure why this is in a separate section here and not with the other includes below...